.genx/templates/angular/route.hbs

```diff
@@ -11,4 +11,22 @@ import {CommonModule} from '@angular/common';
   templateUrl: './{{kebabCase route.name}}.component.html',
   styleUrls: ['./{{kebabCase route.name}}.component.css'],
 })
-export class {{pascalCase route.name}}Component {}
\ No newline at end of file
+export class {{pascalCase route.name}}Component {
+  {{#each route.tilesWithData}}
```

Comment:
@cistov any thoughts on this? I'm on the fance if we should filter out tiles that have config or generate tile with empty config.

Reply:
I would say generate a tile with empty config. The user might just want an empty space to add something they can't do in create?

Reply:
I removed ```tilesWithData``` and left the option for empty tiles:
![image](https://github.com/genesiscommunitysuccess/blank-app-seed/assets/5808873/4710551c-a1b7-4ba6-880b-c6d9b0208a01)